source/_integrations/sms.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: "SMS"
+description: "SMS notification via GSM modem."
+icon: gammu.png
+ha_category:
+  - Notifications
+ha_release: 0.104.4
```

Comment:
We don't release new features in patch releases, furthermore, that version has already been shipped. Maybe just update it to the upcoming version.
```suggestion
ha_release: 0.105
```